docs/01-overview/main-areas/application-connectivity/README.md

```diff
@@ -5,11 +5,11 @@ title: What is Application Connectivity in Kyma?
 Application Connectivity in Kyma is an area that: 
 
 - Simplifies and secures the connection between external systems and Kyma
-- Registers external events and APIs and simplifies the API usage
-- Provides asynchronous communication with services and Functions deployed in Kyma through events
+- Stores and handles the metadata of external systems
+- Provides certificate handling for the [Eventing](../eventing/README.md) flow in the Compass scenario (mode)
 - Manages secure access to external systems
 - Provides monitoring and tracing capabilities to facilitate operational aspects
 
 Depending on your use case, Application Connectivity works in one of the two modes: 
-- **Legacy mode** (default) - using components such as [Application Registry](ac-03-application-registry.md) and [Connector Service](../../../05-technical-reference/00-architecture/ac-02-connector-service.md)
-- **Compass mode** - using [Runtime Agent](ra-01-runtime-agent-overview.md) and integration with [Compass](https://github.com/kyma-incubator/compass)
\ No newline at end of file
+- **Standalone mode** (default) - a standalone mode where Kyma is not connected to [Compass](https://github.com/kyma-incubator/compass)
+- **Compass mode** - using [Runtime Agent](ra-01-runtime-agent-overview.md) and integration with [Compass](https://github.com/kyma-incubator/compass) to automate connection and registration of services using mTLS certificates.
```

Comment:
```suggestion
- **Compass mode** - using [Runtime Agent](ra-01-runtime-agent-overview.md) and integration with [Compass](https://github.com/kyma-incubator/compass) to automate connection and registration of services using mTLS certificates
```

Reply:
... or add a dot in both bullet points 🤔

Reply:
They're not sentences so no dot is fine. Thanks for spotting this :)